models/articles.model.js

```diff
@@ -29,4 +29,38 @@ const selectArticleById = (article_id) => {
   });
 };
 
-module.exports = { selectAllArticles, selectArticleById };
+const updateArticleByArticleId = (article_id, newVote) => {
+  const queryValues = []
+  let sqlString = `UPDATE articles
+  SET votes = votes`
+
+  if(!newVote || typeof newVote != 'number'){
+    return Promise.reject({status: 400, msg: 'bad request'})
+  }
+
+  if (newVote >= 0){
+    queryValues.push(newVote)
+    queryValues.push(article_id)
+    sqlString += ` + $1`
+  }
+
+  else if(newVote < 0){
+    const updatedVote = (Number(newVote.toString().slice(1)))
```

Comment:
I don't see what this is doing here. You'll be passing a whole string to the database which will interpret the number appropriately for you.